README.md

```diff
@@ -23,7 +23,7 @@ case class InitRequest(
     address: Option[InitRequestAddress] = None,
     messages: Option[InitRequestMessages] = None, 
     customisationsUrl: Option[String] = None,
-    directDebitRequirements: Option[InitDirectDebitRequirements] = None,
+    bacsRequirements: Option[InitDirectDebitRequirements] = None,
```

Comment:
Should we change `Option[InitDirectDebitRequirements]` to `Option[InitBACSRequirements]`

Reply:
Yeh - probably - I was just being lazy!